lib/interactivity/event/MouseScrollingEventListener.js

```diff
@@ -0,0 +1,115 @@
+/*
+|--------------------------------------------------------------------------
+| MouseScrollingEventListener
+|--------------------------------------------------------------------------
+|
+| This file defines the MouseScrollingEventListener Object.
+| This package is based on Pixi.js.
+| http://www.pixijs.com/
+|
+*/
+
+const {ScrollingEventListener} = require('./ScrollingEventListener.js');
+const {PixiMouseEventListener} = require('./PixiMouseEventListener.js');
+
+/**
+ * WHEEL
+ * Identify the WHEEL event.
+ * @type {Number} 
+ */
+const WHEEL = 0;
+
+class MouseScrollingEventListener extends ScrollingEventListener
+{
+	/**
+	 * constructor
+	 * This function is used in order to build a MouseScrollingEventListener
+	 * @param {Object2D}  obj  The object2D to bind with the event.
+	 * @param {Number}  code  The code of the event.
+	 * @param {Function}  handle The function called when the event is fired.
+	 * @param {Object}  data  The data using by the handle function.
+	 */
+	constructor(obj, code, handle, data = {})
+	{
+		super(obj, code, handle, data);
+
+		this._handle = handle.bind(this, data);
+
+		let eventListener = new PixiMouseEventListener(obj, PixiMouseEventListener.MOUSE_OVER, this.insideObject2D, this);
+			eventListener = new PixiMouseEventListener(obj, PixiMouseEventListener.MOUSE_OUT, this.outsideObject2D, this);
+	}
+
+	/**
+	 * WHEEL
+	 * @getter
+	 */
+	static get WHEEL()
+	{
+		return WHEEL;
+	}
+
+	/**
+	 * type
+	 * @getter
+	 * This function is a getter for the member _type.
+	 * @return {Number}  The code of this event.
+	 */
+	get type()
+	{
+		return this._type;
+	}
+
+	/**
+	 * target
+	 * @getter
+	 * This function is a getter for the member _target.
+	 * @return {Object2D}  The code of this event.
+	 */
+	get target()
+	{
+		return this._target;
+	}
+
+	/**
+	 * getJSType
+	 * This function is used in order to get the JS type of this event.
+	 * @param {Number}  code  The code of the event.
+	 * @return {String} The JS type as a String.
+	 */
+	getJSType(code)
+	{
+		if ({}.toString.call(code) !== "[object Number]")
+			throw new TypeError("code must be a number.");
+
+		switch(code)
```

Comment:
Why is this switch needed ? If there is only one possible outcome, why using switch ? Same as before, why using the function if the event is always a "wheel" ?